src/BackgroundLayerChooser/BackgroundLayerChooser.tsx

```diff
@@ -88,6 +90,15 @@ export const BackgroundLayerChooser: React.FC<BackgroundLayerChooserProps> = ({
         ovLayer = new OlLayerImage({
           source: selectedLayer.getSource()
         });
+      } else if (selectedLayer instanceof LayerGroup) {
+        if (selectedLayer.get('isVectorTile')) {
```

Comment:
Where is `isVectorTile` coming from? Is this a custom attribute? If so, it needs to be documented.

Reply:
You're right, do you have a suggestion where to document this?

Reply:
Probably it's best to add an appropriate tsdoc entry to the component (that should be catched up by the doc generator/the styleguide or whatever we might want to introduce later on).

Reply:
Documented in https://github.com/terrestris/react-util/pull/478